src/main/java/nl/novi/techiteasycontroller/models/Television.java

```diff
@@ -0,0 +1,237 @@
+package nl.novi.techiteasycontroller.models;
+
+import jakarta.persistence.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "television")
+public class Television {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    Long id;
+    private String type;
+    private String brand;
+    private String name;
+    private Double price;
+    @Column(name = "available_size")
+    private Double availableSize;
+    @Column(name = "refresh_rate")
+    private Integer refreshRate;
+    @Column(name = "screen_type")
+    private String screenType;
+    @Column(name = "screen_quality")
+    private String screenQuality;
+    @Column(name = "smart_tv")
+    private Boolean smartTv;
+    private Boolean wifi;
+    @Column(name = "voice_control")
+    private Boolean voiceControl;
+    private Boolean hdr;
+    private Boolean bluetooth;
+    @Column(name = "ambi_light")
+    private Boolean ambiLight;
+    @Column(name = "original_stock")
+    private Integer originalStock;
+    private Integer sold;
+
+
+    @OneToOne
+    private RemoteController remoteController = new RemoteController();
```

Comment:
de = new RemoteController is hier niet nodig. geldt ook voor  ci module en de wallbrackets hieronder